ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/teleoperation/RDXHumanoidDemoPoses.java

```diff
@@ -140,11 +145,14 @@ private void renderImGuiWidgets()
                delay = Math.max(delay, setFootDown(robotSide, 2.0));
          }
 
+         //         for (RobotSide robotSide : RobotSide.values)
+         armsConfiguration.put(RobotSide.LEFT, leftArmHomeConfiguration);
+         armsConfiguration.put(RobotSide.RIGHT, rightArmHomeConfiguration);
          for (RobotSide robotSide : RobotSide.values)
-            appendArmTrajectoryMessageToPublish(robotSide, armsConfiguration.get(robotSide), teleoperationParameters.getTrajectoryTime(), 0.1);
+            appendArmTrajectoryMessageToPublish(robotSide, armsConfiguration.get(robotSide), 3.0, delay);
 
-         appendChestOrientationToPublish(chestOrientation, teleoperationParameters.getTrajectoryTime(), 0.1);
-         appendPelvisOrientationToPublish(pelvisOrientation, pelvisPosition, teleoperationParameters.getTrajectoryTime(), 0.1);
+         appendChestOrientationToPublish(chestOrientation, 3.0, 0.1);
+         appendPelvisOrientationToPublish(pelvisOrientation, pelvisPosition, 3.0, 0.1);
```

Comment:
are these hard-coded trajectory durations? Can we pull that into a static variable at the top of the calss?